bpf/lib/nat.h

```diff
@@ -1393,6 +1608,7 @@ snat_v6_rev_nat(struct __ctx_buff *ctx, const struct ipv6_nat_target *target)
 		__be16 dport;
 	} l4hdr;
 	__u32 off;
+	__u8 nexthdr;
```

Comment:
Cosmetic nit - we could now have this inside the ICMPV6_PKT_TOOBIG case. Maybe worth cleaning up in a follow-on PR?

Reply:
Yes I will do that once merged! Thank you for your great reviews on this change.

Reply:
#24537